crates/astria-sequencer/justfile

```diff
@@ -28,18 +28,18 @@ run-testnet:
 
 [linux]
 stop-testnet:
-  pkill cometbft && pkill astria-sequencer
+  pkill cometbft && pkill -f astria-sequencer
 [linux] 
 run-testnet: 
   cargo build
-  ${OUT_DIR} ${NUM_VALIDATORS} ./scripts/testnet.sh
+  ./scripts/testnet.sh
 
 cometbft-logs-testnet NODE="node0":
   tail -f sequencer_testnet/{{ NODE }}/cometbft.log
 
 sequencer-logs-testnet NODE="node0":
   tail -f sequencer_testnet/{{ NODE }}/sequencer.log
 
-clean-testnet DIR="sequencer_testnet": stop-testnet
+clean-testnet DIR="sequencer_testnet": 
```

Comment:
As mentioned above, if we ignore the results of `pkill`, we could reinstate this, since it seems like we wouldn't want to clean the dirs if the testnet is still running?

Reply:
re-added it!